examples/gno.land/r/gov/dao/memberset.gno

```diff
@@ -0,0 +1,40 @@
+package govdao
+
+import (
+	"std"
+
+	pproposal "gno.land/p/gov/proposal"
+)
+
+const daoPkgPath = "gno.land/r/gov/dao"
+
+const (
+	errNoChangesProposed = "no set changes proposed"
+	errNotGovDAO         = "caller not govdao executor"
+)
+
+func NewPropExecutor(changesFn func() []std.Address) pproposal.Executor {
```

Comment:
Just realized this proposal only supports member additions, but not member removals.

Maybe we should wait for #2380 to give us the callable ability of govdao without the custom executor?

Reply:
The whole DAO depends on this Executor to make it work. AFAIK both methods are not exclusive. Maybe we can keep this as it is now and when #2380 is merged, implement both methods if needed.